### DISCUSSION TITLE
Data online without local server is possible?

### DISCUSSION THREAD

**Discussion by komax74** [jomjol/AI-on-the-edge-device#1163] [Q&A]
I discovered this project and it's spectacular, congratulations to the author and the whole community.
I've been searching but not found any information to be able to write data ( or read it ) directly online without need to create a local server for home assistant, node-red or similar.
Is there any way to create a dashboard with consumption on cloud platform like google sheet, blynk, make or others?
Thank you very much.

**Comment by caco3** [comment]
You would need to get the data to the internet somehow.
Using the current functionality, the only safe way I see is to use a public MQTT broker and a service to receive/visualize it.
The unsafe way would be to make it accessible over the internet and call http://<IP>/json periodically :bangbang: Highly unsafe :bangbang: 

If somebody is willing to implement it, the best approach would be to add a configurable URL which would be called by the device after each run to upload the new data.
